src/components/AvatarWithDisplayName.js

```diff
@@ -88,6 +92,22 @@ function AvatarWithDisplayName(props) {
                             textStyles={[props.isAnonymous ? styles.headerAnonymousFooter : styles.headerText, styles.pre]}
                             shouldUseFullTitle={isExpenseReport || props.isAnonymous}
                         />
+                        {!_.isEmpty(subtitleLink) && (
+                            <PressableWithoutFeedback
+                                onPress={() => {
```

Comment:
I'll keep this as is for now since it's mostly the same logic for `subtitle` few lines below. I think if we ever change this to useCallback then we should change it for both. Let me know if you feel strongly about changing this now.